src/Rooms.ts

```diff
@@ -28,7 +29,18 @@ import { MatrixClientPeg } from './MatrixClientPeg';
  * @returns {string} A display alias for the given room
  */
 export function getDisplayAliasForRoom(room: Room): string {
-    return room.getCanonicalAlias() || room.getAltAliases()[0];
+    return getDisplayAliasForAliasSet(
+        room.getCanonicalAlias(), room.getAltAliases(),
+    );
+}
+
+// The various display alias getters all feed through this one path so there's a
```

Comment:
because time is a thing in our project:
```suggestion
// The various display alias getters should all feed through this one path so there's a
```

Reply:
Haha, thanks, added.